test/models/history_test.rb

```diff
@@ -85,7 +85,7 @@ def history_types
     end
 
     assert_difference("History.in_time_frame('Last 14 Days').size", 1) do
-      create(:history, history_type: 'Comment').update(created_at: 1.day.ago)
+      create(:history, history_type: 'Comment').update(created_at: Date.today.to_datetime - 1.hour)
```

Comment:
out of curiosity, how does this change fix this test?

Reply:
It turns out that `Date.today.to_datetime` will actually take the current date in you system's local time, then `to_datetime` will return it as a DateTime that it SAYS is in UTC but may not actually be the current date in UTC. For example, if it's 9PM UTC-6 and you run `Date.today.to_datetime`, then it will return `Tue, 13 Jul 2021 00:00:00 +0000` when in UTC it should actually be `Wed, 14 Jul 2021 00:00:00 +0000`. 

I'm going to change the scope so that it always uses UTC regardless of the local system's time zone